packages/meditrak-app/app/assessment/specificQuestions/Instruction.js

```diff
@@ -65,7 +65,7 @@ const localStyles = StyleSheet.create({
   textMain: {
     fontSize: THEME_FONT_SIZE_THREE,
     lineHeight: getLineHeight(THEME_FONT_SIZE_THREE),
-    fontWeight: '500',
+    fontWeight: 'bold',
```

Comment:
Seems like there's a bug with numeric `fontWeight` in react-native (`https://github.com/facebook/react-native/issues/26193`). `bold` is the only value that works...